docs/setup_installation/aws/getting_started.md

```diff
@@ -5,9 +5,32 @@ in the cloud. It integrates seamlessly with third-party platforms such as Databr
 SageMaker and KubeFlow. This guide shows how to set up [managed.hopsworks.ai](https://managed.hopsworks.ai) with your organization's AWS account.
 
 ## Prerequisit
-To run the commands in this guide, you must have the AWS CLI installed and configured. See the [Getting started guide](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-install.html) in the AWS CLI User Guide for more information.
-
-All the commands have unix-like quotation rules. These commands will need to be adapted to your terminal's quoting rules. See [Using quotation marks with strings](https://docs.aws.amazon.com/cli/v1/userguide/cli-usage-parameters-quoting-strings.html) in the AWS CLI User Guide .
+To run the commands in this guide, you must have the AWS CLI installed and configured and your user must have at least the set of permission listed below. See the [Getting started guide](https://docs.aws.amazon.com/cli/v1/userguide/cli-chap-install.html) in the AWS CLI User Guide for more information about installing and configuring the AWS CLI.
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "VisualEditor0",
+            "Effect": "Allow",
+            "Action": [
+                "iam:CreateInstanceProfile",
+                "iam:PassRole",
```

Comment:
I guess you have tried to setup an account with a user with these permissions, right?

just out of curiosity does this mean that even while the user doesn't have the permissions listed on Hopsworks cross account rule, he can pass our role to access the cluster based on the fact that he allows `iam:PassRole`

Reply:
Yes. I have started from a user without any permissions and added the permissions one by one as they were needed.
I think it is more the create role that mater. If you can create a role you can give it the permissions you want.